CIP-????/README.md

```diff
@@ -0,0 +1,250 @@
+    - https://github.com/cardano-foundation/CIPs/pull/102
+Created: 2023-03-21
+License: CC-BY-4.0
+---
+
+## Abstract
+
+The Cardano Foundation proposes a mechanism for polling Cardano stake pool operators on specific topics. Polls are done on-chain through transaction metadata and authenticated through stake pool credentials (either VRF public key similar to what's described in [CIP-0022](../CIP-0022) or Ed25519 cold key). The goal is to gather opinions on governance matters such as protocol parameter updates. This standard is an inclusive interim solution while the work on a larger governance framework such as [CIP-1694][] continues.
+
+## Motivation: why is this CIP necessary?
+<!-- A clear explanation that introduces the reason for a proposal, its use cases and stakeholders. If the CIP changes an established design, then it must outline design issues that motivate a rework. For complex proposals, authors must write a Cardano Problem Statement (CPS) as defined in CIP-9999 and link to it as the `Motivation`. -->
+
+Governance is difficult. Discussions on CIP-1694 can attest to that quite clearly. There are constant debates within the Cardano community about changing protocol parameters, and the decision ultimately falls -- at this stage still -- onto the three genesis entities: Input Output, The Cardano Foundation and Emurgo. Yet, at this stage, few governance tools are at their disposal to make educated decisions. Besides Twitter polls, newsletter surveys, and SPO town halls on Discord, we have identified a gap and an opportunity to engage with the Cardano community through means currently at our disposal.
+
+Conducting an on-chain poll between SPOs can also be seen as an experiment and an evaluation of the network's participation and engagement in the governance questions. Even though we only propose to poll one particular group of the Cardano community (the SPOs), such events can help to provide actual data to fuel the conversations around CIP-1694.
+
+In summary, the goals are:
+
+1. [x] to make some first experimental baby steps in the realm of governance;
+1. [x] to capture participation data from SPOs;
+1. [x] to raise awareness amongst SPOs regarding their future role in governance;
+1. [x] to keep the Voltaire dynamics up in the ecosystem while other efforts are being pursued;
+1. [x] to improve relations between the Cardano Foundation & SPOs for better mutual understanding and fruitful conversations.
+
+
+## Specification
+
+### Overview
+
+Polls will be multiple-choice questions by The Cardano Foundation with pre-defined answers to choose from.
+
+Here's an example of a question and answers:
+
+- _Pineapples on pizza?_
+  - [ ] yes
+  - [ ] no
+
+The serialised question and answers will be posted on-chain and signed by one of the delegate genesis keys owned by The Cardano Foundation. Answers will be provided on-chain by participating SPOs via transaction metadata referring to:
```

Comment:
What about adding a pre-emptive description about how this might be done after CIP-1694, or something like it, removes the Genesis keys?  https://github.com/JaredCorduan/CIPs/blob/voltaire-v1/CIP-1694/README.md#changes-to-the-transaction-body

Reply:
I believe, if I'm not mistaken, that this tool is only intended as a stop-gap until the Voltaire era kicks off and so would become redundant based on having the concept of a "poll" in CIP-1694. Also, interested third parties who wish to conduct polling of SPOs could add some form of key registration (non-Genesis key) in the future as an expansion to this CIP to provide "vote certification provenance".

Reply:
Indeed. CIP-1694 would technically render this CIP obsolete especially now that we also added a "Poll" governance action to it. I should perhaps make that clearer in the CIP title that this is intended to be an interim solution only.